templates/corporate/upgrade.html

```diff
@@ -171,7 +171,7 @@ <h1>$<span class="due-today-price"></span></h1>
                             </span>
                             <object class="loader upgrade-button-loader" type="image/svg+xml" data="{{ static('images/loading/loader-white.svg') }}"></object>
                         </button>
-                        <div id="autopay-error" class="alert alert-danger upgrade-page-error"></div>
+                        <div id="autopay-error" class="alert alert-danger upgrade-page-error hide"></div>
                     </div>
                 </form>
                 {% if error_message %}
```

Comment:
Is this other `error_message` block in this page dead code?

Reply:
yeah, seems like it has been dead for a long time.

Reply:
Added a commit to remove it.